build-logic/android-convention/src/main/kotlin/convention.android-base.gradle.kts

```diff
@@ -31,4 +30,16 @@ configure<BaseExtension> {
         textReport = true
         isQuiet = true
     }
+
+    @Suppress("UnstableApiUsage")
+    with(buildFeatures) {
+        aidl = false
+        compose = false
+        buildConfig = false
+        prefab = false
+        renderScript = false
+        resValues = false
+        shaders = false
+        viewBinding = false
+    }
```

Comment:
Just thoughts out loud. 
We have defaults:

```properties
android.defaults.buildfeatures.aidl=false
android.defaults.buildfeatures.buildconfig=false
...
```
But it's easy to miss renamings. Configuration by code is more robust.